### PR TITLE
Add ability to override DockerClientFactory.TINY_IMAGE in testcontainers.properties

### LINKED ISSUE #281: Strong coupling with "alpine:3.2" image in DockerClientFactory
Hello.

When #267 was merged running docker inside docker works fine :)  This feature is very useful for CI pipelines when Jenkins slave executes in Docker container, for example.

But build servers may have partial Internet connectivity  - to the local Docker registry only (just like in my case). So despite of wonderful docker-in-docker support build fails nevertheless on DockerClientFactory.instance().client() invoke.

Can we get rid of this builtin image dependency or make it customizable at least? For example, pass image name  to DockerClientFactory through environment variable?

### PR DESCRIPTION
**Pull request by dddpaul**
This fixes #281.

### CODE REVIEW

File: core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java [hunk 1]
```diff
@@ -42,6 +43,7 @@ private static TestcontainersConfiguration loadConfiguration() {
 
                 config.ambassadorContainerImage = properties.getProperty("ambassador.container.image", config.ambassadorContainerImage);
                 config.vncRecordedContainerImage = properties.getProperty("vncrecorder.container.image", config.vncRecordedContainerImage);
+                config.tinyImage = properties.getProperty("tinyimage.container.image", config.vncRecordedContainerImage);
```
Comment (bsideup) [inline core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: default value is wrong. should be config. tinyImage

Comment (dddpaul) [inline core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: Sorry, it's friday night :) Fixed.

**Comment by bsideup** [comment]
LGTM

/cc @rnorth

**Comment by rnorth** [comment]
Great! Thanks - this looks good to me. Will merge.

**Pull request merged**